.github/workflows/test-runner-on-perf.yml

```
@@ -8,14 +8,40 @@ env:
   GOOGLE_ZONE: us-central1
   K8_CLUSTER: jade-master-us-central1
 on:
-  schedule:
-    - cron: '0 4 * * *' # run at 4 AM UTC, 12PM EST.
+  #schedule:
+  #  - cron: '0 4 * * *' # run at 4 AM UTC, 12PM EST.
+  workflow_dispatch: {}
 jobs:
   test-runner-perf:
     runs-on: ubuntu-latest
     steps:
       - name: "Checkout code"
         uses: actions/checkout@v2
+      - name: "Import Vault perf secrets"
+        uses: hashicorp/vault-action@c8b383ee
```

```suggestion
        uses: hashicorp/vault-action@v2.0.1
```

Hashicorp stabilized their vault action.